src/flask_track_usage/__init__.py

```diff
@@ -184,6 +188,8 @@ def after_request(self, response):
         }
         if ctx.request.authorization:
             data['username'] = str(ctx.request.authorization.username)
+        elif getattr(self.app, 'login_manager', None):
+            data['username'] = str(current_user)
```

Comment:
If Flask-Login is in use, but they are not logged in I fear this will end up storing a user named "None" in some cases. Example with SQLAlchemy:

```python
>>> import sqlalchemy
>>> from sqlalchemy import Column, String
>>> c = Column('test', String(255))
>>> c.nullable
>>> True  # <-- If the field gets a NoneType of None it will be null. If it's a string it will store it as such.
```
Looking at `current_user`:
```python
>>> from flask_login import current_user
>>> current_user
None
>>> str(current_user)
'None' # <-- Note the NoneType has become a str with the value of None
```

I _think_ this may do the trick:
```suggestion
            if current_user is not None:
                data['username'] = str(current_user)
```

Reply:
Maybe it is returning None because you haven't attached it to a Flask app. In my case, when user is not logged yet, it returns an Anonymous object. It is true that I thought that it will saved as "Anonymous", but a quick test proved me wrong.

```python
> /dir-to-venv/venv8/lib/python3.8/site-packages/flask_track_usage/__init__.py(193)after_request()
-> data['username'] = str(current_user)
(Pdb) current_user
<flask_login.mixins.AnonymousUserMixin object at 0x7f1ee5fe8580>
(Pdb) str(current_user)
'<flask_login.mixins.AnonymousUserMixin object at 0x7f1ee5fe8580>'
(Pdb) dir(current_user)
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', 'get_id', 'is_active', 'is_anonymous', 'is_authenticated']
(Pdb) current_user.is_anonymous
True
(Pdb) 
```

So I will update the PR to check both cases, that current_user is not none and if user is anonymous save it as "Anonymous" or do not save it if you prefer that way